local_store.go

```diff
@@ -218,25 +221,66 @@ func (f *fileSystemStore) stagedDir() string {
 	return filepath.Join(f.dir, "staged")
 }
 
+func isMetaFile(e os.DirEntry) (bool, error) {
+	name := e.Name()
+	if e.IsDir() || !(filepath.Ext(name) == ".json" && roles.IsTopLevelManifest(name)) {
```

Comment:
Assuming that you've noted that this is only for top-level manifests and will generalize to delegated roles by removing the final conditional?

Reply:
Correct. In https://github.com/theupdateframework/go-tuf/pull/175, this line is changed to

```go
	if e.IsDir() || filepath.Ext(e.Name()) != ".json" {
```

Reply:
Thanks Ethan! Always thorough